docs/hardware.rst

```diff
@@ -84,64 +84,68 @@ Workstations
-      Authenticator <https://support.google.com/accounts/answer/1066447?hl=en>`__)
-   -  A dedicated hardware dongle that computes HOTP codes (e.g. a
-      `YubiKey <https://www.yubico.com/products/yubikey-hardware/yubikey/>`__).
-
--  **Transfer Device(s)**: You need a mechanism to transfer encrypted
-   submissions from the **Journalist Workstation** to the *SVS* to
-   decrypt and view them. The most common transfer devices are DVD/CD-R
-   discs and USB drives.
-
-   -  From a security perspective, it is preferable to use write-once
-      media such as DVD/CD-R discs because it eliminates the risk of
-      exfiltration by malware that persists on the Transfer Device (e.g.
-      `BadUSB <https://srlabs.de/badusb/>`__).
-   -  On the other hand, using write-once media to transfer data is
-      typically inconvenient and time-consuming. You should consider
-      your threat model and choose your transfer device accordingly.
-
--  **Monitor, Keyboard, Mouse**: You will need these to do the initial
-   installation of Ubuntu on the *Application* and *Monitor Servers*.
-
-   -  Depending on your setup, you may also need these to work on the
-      *SVS*.
+Secure Viewing Station (SVS)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1 physical computer used as an airgap to decrypt and view submissions retrieved
+from the *Application Server*.
+
+The chosen hardware should be solely used for this purpose and should have any
+wireless networking hardware removed before use.
+
+Admin/Journalist Workstation(s)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+*At least 1* physical computer that is used as a workstation for SecureDrop
+admins and/or journalists.
+
+Each Admin and Journalist will have their own bootable Tails USB with an
+encrypted persistent partition that they will use to access SecureDrop. You will
+need at least one *workstation* to boot the Tails USBs, and may need more
+depending on: the number of admins/journalists you wish to grant access to
+SecureDrop, whether they can share the same workstation due to availability
+requirements, geographic distribution, etc.
+
+USB drive(s)
+~~~~~~~~~~~~~~~~
+
+*At least 2* USB drives to use as a bootable Tails USB for the *SVS* and the
+**Admin Tails**/**Journalist Tails**.
+
+If only one person is maintaining the system, you may use the same Tails
+instance as both the Admin Tails and the Journalist Tails; otherwise, we
+recommend buying 1 drive for each admin and each journalist.
+
+We also recommend buying two additional USBs to use as bootable backups of the
+*SVS* and **Admin Tails**.
+
+**Two-factor authenticator**: Two-factor authentication is used when connecting
+to different parts of the SecureDrop system. Each admin and each journalist
+needs a two-factor authenticator. We currently support two options for t
+wo-factor authentication:
+
+* Your existing smartphone with an app that computes TOTP codes
+  (e.g. `Google Authenticator <https://support.google.com/accounts/answer/1066447?hl=en>`__).
```

Comment:
And FreeOTP, which is more in line with the free/ open-source software policies of the rest of the project.